games/s.yaml

```diff
@@ -74,6 +74,21 @@
     lang: JavaScript
     license: [MIT]
 
+- name: The Sims Online
+  meta:
+    genre: [MMORPG]
+    subgenre: [Life]
+    theme: [Management]
+  remakes:
+    - name: Project Dollhouse
+      repo: https://github.com/Afr0Games/Project-Dollhouse
+      url: http://www.afr0games.com
+      development:
```

Comment:
No value set for `development` is what's breaking the build. Remove the key or add a value to it. 😌